code/datums/ai/monkey/monkey_subtrees.dm

```diff
@@ -41,7 +41,7 @@
 	var/datum/weakref/target_ref = controller.blackboard[BB_MONKEY_CURRENT_ATTACK_TARGET]
 	var/mob/living/selected_enemy = target_ref?.resolve()
 
-	if(!selected_enemy.stat) //He's up, get him!
+	if(selected_enemy && !selected_enemy.stat) //He's up, get him!
```

Comment:
//by this point we have a target but they're down, let's try dumpstering this loser

Right below this. Do we still need to go through all the processing if target is null?

Reply:
Good point, I'll just have it abort the subtree

Reply:
The new version ends combat and the subtree if the enemy is null.